test/lib/time.rb

```diff
@@ -22,20 +29,27 @@ def convert_fragment(input)
 
   attr_reader :sort_middle
 
-  def hour_conversion(input, minutes_array)
+  def check_input(input)
+    mistakes = [' ', [['']]]
+    mistakes.each do |mistake|
+      raise(ArgumentError) if input == mistake || input.nil? || input.empty?
```

Comment:
Are you sure you need to raise exception?

Reply:
I am not sure. But I don't know which command I can use to replace abort.

Reply:
you can change your code and use return to finish code execution